libs/pruviloj/Pruviloj/Core.idr

```diff
@@ -268,3 +274,37 @@ reflexivity =
                 , NamePart `{reflexivity}
                 , TextPart "is not applicable."
                 ]
+
+||| Attempt to swap the sides of equality in a goal.
+symmetry : Elab ()
+symmetry = do
+    [_,_,_,_,_] <- apply (Var `{{sym}}) [True, True, True, True, False]
+       | _ => fail [TextPart "Failed while applying", NamePart `{symmetry} ]
+    solve
+    attack
```

Comment:
The first `solve` here introduces a `pi_arg`, so `attack` makes it into hole in case you need a subsequent call to `rewriteWith`. As I said, I'm not too sure about `symmetry` tho.